tests/schemas/invalid/test_invalid_mismatching_answer_label_and_value.json

```diff
@@ -0,0 +1,130 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "023",
+  "title": "Multiple answers per question",
+  "description": "Check for mismatching answer labels and values",
+  "theme": "default",
+  "legal_basis": "Notice is given under section 1 of the Statistics of Trade Act 1947.",
+  "metadata": [
+    {
+      "name": "user_id",
+      "type": "string"
+    },
+    {
+      "name": "period_id",
+      "type": "string"
+    },
+    {
+      "name": "ru_name",
+      "type": "string"
+    }
+  ],
+  "sections": [
+    {
+      "id": "default-section",
+      "groups": [
+        {
+          "blocks": [
+            {
+              "type": "Question",
+              "id": "personal-details-block",
+              "question": {
+                "answers": [
+                  {
+                    "id": "first-name-answer",
+                    "label": "What is your first name",
+                    "mandatory": false,
+                    "q_code": "1",
+                    "type": "TextField"
+                  },
+                  {
+                    "id": "surname-answer",
+                    "label": "What is your surname",
+                    "mandatory": false,
+                    "q_code": "2",
+                    "type": "TextField"
+                  }
```

Comment:
This schema could be simplified further:
- Single name field in first block
- Drop `q_code` properties
- Can the `tea-or-coffee` block be dropped and the `No` answer label for the `correct` block be changed to not match the label?

Reply:
Have changed the validator so that it checks for dynamic labels and have updated the schema.